web-components/src/components/usa-identifier/index.js

```diff
@@ -0,0 +1,156 @@
+      this.domain.classList.add("usa-identifier__identity-domain");
+    }
+    /**
+     * Scaffold disclaimer text:
+     * Add necessary classes for styling
+     * Wrap "An" in aria-hidden span
+     */
+    if (this.disclaimer) {
+      this.disclaimer.classList.add("usa-identifier__identity-disclaimer");
+    }
+    /**
+     * For English implementations, wrap "An" in an aria-hidden span
+     * This prevents "An official" from sounding like "Unofficial" in audible readouts
+     */
+    // if (this.disclaimer.innerHTML.includes("An official")) {
+    //   this.disclaimer.innerHTML = this.disclaimer.innerHTML.replace(
+    //     "An official",
+    //     '<span aria-hidden="true">An</span> official'
+    //   );
+    // }
+
+    if (this.disclaimer || this.domain) {
+      return html`
+        <section
+          class="usa-identifier__identity"
+          aria-label="Agency description"
+        >
+          ${this.domain} ${this.disclaimer}
+        </section>
+      `;
+    }
+  }
+
+  // Render the logos and text in the masthead
+  mastheadTemplate() {
+    if (this.domain || this.disclaimer || this.logos.length > 0) {
+      return html`
+        <section
+          class="usa-identifier__section usa-identifier__section--masthead"
+          aria-label="Agency identifier"
+        >
+          <div class="usa-identifier__container">
+            ${this.mastheadLogosTemplate()} ${this.mastheadTextTemplate()}
+          </div>
+        </section>
+      `;
+    }
+  }
+
+  // Render the list of links
+  linksTemplate() {
+    if (this.links && this.links.length > 0) {
+      return html`
+        <nav
+          class="usa-identifier__section usa-identifier__section--required-links"
+          aria-label="Important links"
+        >
+          <div class="usa-identifier__container">
+            <div class="usa-identifier__logos">
+              <ul class="usa-identifier__required-links-list">
+                ${this.links.map((link) => {
+                  link.classList.add("usa-identifier__required-link");
+                  return html`<li class="usa-identifier__required-links-item">
+                    ${link}
+                  </li>`;
+                })}
+              </ul>
+            </div>
+          </div>
+        </nav>
+      `;
+    }
+  }
+
+  // Render the footer USA.gov text
+  usagovTemplate() {
+    /**
+     * Scaffold usagov text:
+     * Add necessary classes for styling
+     */
+    if (this.usagov) {
+      const usagovLink = this.usagov.querySelector("a");
+      this.usagov.classList.add("usa-identifier__usagov-description");
+      usagovLink.classList.add("usa-link");
+
+      return html`
+        <section
+          class="usa-identifier__section usa-identifier__section--usagov"
+        >
+          <div class="usa-identifier__container">
+            <div class="usa-identifier__usagov-description">${this.usagov}</div>
+          </div>
+        </section>
+      `;
+    }
+  }
```

Comment:
This seems like it should be included by default.

Reply:
I updated the build to include the usa gov section by default and added default content for `es`/`en` lang implementations. Now, users don't need to include the usagov slot in their implementation unless they need to customize. See the docs for the "Custom translation" story for an example of this.